packages/block-library/src/calendar/block.json

```diff
@@ -16,7 +16,14 @@
 		}
 	},
 	"supports": {
-		"align": true
+		"align": true,
+		"color": {
+			"link": true,
+			"background": false,
```

Comment:
The table block supports background color, should it also support it in the calendar block?
In my opinion, I think support is unnecessary.

Reply:
I think it should be enabled because it would give everyone more design options.

Reply:
I agree

Reply:
Sorry, late to the party. I had been toying around with adding color support for the calendar block over in https://github.com/WordPress/gutenberg/pull/43299

I'll close my PR in favour of this one. 

Just wanted to mention something I discovered in relation to background color support:

When selecting a preset color, followed by a custom color, the block doesn't rerender.

I've pinpointed it to the use of [useDebounce in the ServerSideRender component](916abc45e6682f4bdb70888aa41e98aa395/packages/server-side-render/src/server-side-render.js#L145).

The debounced function is called, but it doesn't fire. I suspect that it's cancelled too early [here](https://github.com/WordPress/gutenberg/blob/33d84b036592a5bf31af05b7710f3b2b14163dc4/packages/compose/src/hooks/use-debounce/index.js). 

It's a strange equality check bug that I haven't worked out yet.

I tested on this branch:

![2022-08-19 12 08 58](https://user-images.githubusercontent.com/6458278/185527428-014efa9f-370a-4f38-ba96-8df5d1b769da.gif)

It occurs every so often, but not all the time so might be acceptable.

But if I enable`"gradients": true` in color supports, the effect is more pronounced:

https://user-images.githubusercontent.com/6458278/185527648-a6b6c547-11f9-47a3-8c18-e315dde1c83b.mp4

Lesson: don't enable "gradient" for now 😄